packages/tocco-util/src/notifier/notificationActionFactory.js

```diff
@@ -90,15 +97,15 @@ export function getYesNoAction(title, message, yesText, noText, cancelText, onYe
 
 export function getBlockingInfo(id, title, message, icon) {
   const options = {
-    timeOut: 0,
-    showCloseButton: false,
     attention: true,
+    component: () => <TitleMessage title={title} message={message}/>,
+    icon: icon ? <Icon icon={icon} size="3x" /> : <IconTocco size="3em" />,
```

Comment:
Default Icon as in info would be good. (e.g. circle-notch oder toco loader?)
A blocking info always indicates, that something is running. So imo this Icon should Spin.